src/main/java/de/tum/in/www1/artemis/service/metis/ReactionService.java

```diff
@@ -18,6 +18,8 @@ public class ReactionService {
 
     private static final String METIS_REACTION_ENTITY_NAME = "posting reaction";
 
+    private static final String VOTE_EMOJI_ID = "heavy_plus_sign";
```

Comment:
Maybe add a comment that this constant is the same in the client at metis.util.ts#28.

Reply:
Done ✔️